### PR TITLE
Followup edits to PR 10312

### PR DESCRIPTION
**Pull request by bfallonf**
For: #10312 

cc @JacobTanenbaum 

Fixing the errant ==== and some word changes.

Didn't go all out on this because of the upcoming 4.0 efforts. 

@openshift/team-documentation PTAL

### CODE REVIEW

File: install_config/configuring_sdn.adoc [hunk 1]
```diff
@@ -159,12 +159,14 @@ but not to *172.28.0.0/14*, because even though the original range is entirely
...
 
+[[config-changing-vxlan-port-for-cluster-network]]
+== Changing the VXLAN PORT for the cluster network
 
-.Changing the VXLAN PORT for the cluster network
-
-A cluster administrator can change the VXLAN port the system uses by changing the vxlanPort variable in the
-master configuration file.
+As a cluster administrator, you can change the VXLAN port the system uses by
+changing the `vxlanPort` variable in the master configuration file at
```
Comment (kalexand-rh) [inline install_config/configuring_sdn.adoc]: I'd s/file at/file, located in the <dir> directory
You should probably at least s/file at/file, located in

File: install_config/configuring_sdn.adoc [hunk 2]
```diff
@@ -180,16 +182,14 @@ networkConfig:
...
+<1> Set to the value used by the nodes for the VXLAN Port. It can be an integer between 1-65,535. The default value is `4789`.
 
 [IMPORTANT]
 ====
-An administrator cannot change the VXLAN port of a running clusternetwork object. In order to change a cluster's VXLAN port
-it is required to delete the running clusternetwork, make the change in the master configuration file restart, the master and then
-restart all other nodes to fully propogate the change.
-====
-
+You cannot change the VXLAN port of a running `clusternetwork` object. In order
```
Comment (kalexand-rh) [inline install_config/configuring_sdn.adoc]: s/In order to/To

File: install_config/configuring_sdn.adoc [hunk 3]
```diff
@@ -180,16 +182,14 @@ networkConfig:
   serviceNetworkCIDR: 172.30.0.0/16
   vxlanPort: 4789 <1>
 ----
-<1> vxlanPort is the value used by the nodes for the VXLAN Port. It can be an integer between 1-65,535, if unspecified
-by the cluster administrator the default value is 4789.
+<1> Set to the value used by the nodes for the VXLAN Port. It can be an integer between 1-65,535. The default value is `4789`.
```
Comment (kalexand-rh) [inline install_config/configuring_sdn.adoc]: Style says to omit the comma in 65,535

I'd say something like "The VXLAN port number that the nodes use. Enter an integer in the range 1-65535." but :man_shrugging:

**Review by kalexand-rh — COMMENTED** [review]
Style pick, couple of suggestions. It looks pretty good!

**Comment by bfallonf** [comment]
Thanks @kalexand-rh . I've made changes.

@zhaozhanqi I have updated this PR with your comments from #10312 , but changed it to in the procedure rather than in the IMPORTANT box. Please let me know if you agree with the changes.

**Comment by vikram-redhat** [comment]
@zhaozhanqi - can you please take a look?

**Comment by zhaozhanqi** [comment]
better remind customer to add the iptables rule for new port on all nodes.
Other LGTM

**Comment by vikram-redhat** [comment]
@kalexand-rh - if you are looking for a small fix, this should be ready to merge after incorporating the last comment from @zhaozhanqi . You could do that small fix in a follow up PR.

**Comment by kalexand-rh** [comment]
@vikram-redhat, I'll be sure to get this change and its follow-up in before we release 3.11.

**Pull request merged**